qutip/qip/noise.py

```diff
@@ -338,9 +274,9 @@ def get_noisy_dynamics(self, dims):
                     T2_eff = 1./(1./t2-1./2./t1)
                 else:
                     T2_eff = t2
-                op = 1/np.sqrt(2*T2_eff) * sigmaz()
-                lindblad_noise.add_lindblad_noise(op, qu_ind, coeff=True)
-        return lindblad_noise
+                op = 1/np.sqrt(2*T2_eff) * 2 * num(dims[qu_ind])
```

Comment:
`sigmaz()` and `2 * num(2)` aren't equivalent operators - I assume the constant offset doesn't matter so much, but `num` will also "swap" the effects on the two basis states?  compare `[[1, 0], [0, -1]]` to `[[0, 0], [0, 2]]`.  I don't know if that's a problem for you, though.

Reply:
You are right that it swaps the energy level. It will probably cause the state to rotate in a different direction? In my case, the measured T2 time is not changed since the tests pass. If one looks closer, there will be difference.

Since in qip the cavity is modelled with the number operator, I think `num` should be the more correct choice here.

Reply:
ah ah, I was thinking about the second one as like an amplitude-damping term, where really it's a dephasing, right?  This is a long way out of my area of expertise, but if I remember correctly, the derivation of dephasing Kraus operators considers the effect of phase rotation by a normally-distributed angle, right?  If that's the case, then this is identical - the distribution is symmetric around the mean rotation of 0, so it doesn't matter which sign you define the operator with